### PR TITLE
separate static configuration

### PR DESCRIPTION
**Pull request by muhammad-ammar**
This will move the static configuration into its own yaml file. We need this to avoid updating ansible role every time we introduce a new variable. From now onwards please add all static configuration in `static_config.yaml`

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@muzaffaryousaf @mushtaqak @yro @Qubad786 FYI

File: instance_config.yaml [hunk 1]
```diff
@@ -136,66 +121,3 @@ sg_script_key:
...
-    hls:
-        - hls
-
-#--
-# Heal settings
-heal_start: 1
-heal_end: 144
-global_timeout: 40
-
-...
```
Comment (mushtaqak) [inline instance_config.yaml]: Three dots mystery 😆

Comment (Qubad786) [reply instance_config.yaml]: finally resolved.

File: VEDA/utils.py [hunk 2]
```diff
@@ -61,14 +62,17 @@ def get_config(yaml_config_file=DEFAULT_CONFIG_FILE_NAME):
...
             yaml_config_file
         )
 
     with open(yaml_config_file, 'r') as config:
         config_dict = yaml.load(config)
 
-    return config_dict
+    with open(STATIC_CONFIG_FILE_PATH, 'r') as config:
+        static_config_dict = yaml.load(config)
+
```
Comment (mushtaqak) [inline VEDA/utils.py]: I think, there should be a comment here.

File: VEDA/utils.py [hunk 3]
```diff
@@ -8,8 +8,9 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 
-DEFAULT_CONFIG_FILE_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+CONFIG_FILE_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```
Comment (Qubad786) [inline VEDA/utils.py]: CONFIG_ROOT_DIR would be better? As currently, its giving an impression as if it is config file path but its config dir path.

**Pull request merged**